<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Level</class>
 <widget class="QWidget" name="Level">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Level</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Level
{
    border-image: url(:/new/prefix1/pic2.png);
}</string>
  </property>
  <widget class="QRadioButton" name="radioButton_1">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
color:rgb(255, 255, 255);
	
	font: 75 24pt &quot;黑体&quot;;
}</string>
   </property>
   <property name="text">
    <string>简单</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
color:rgb(255, 255, 255);
	
	font: 75 24pt &quot;黑体&quot;;
}</string>
   </property>
   <property name="text">
    <string>中等</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>440</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
color:rgb(255, 255, 255);
	
	font: 75 24pt &quot;黑体&quot;;
}</string>
   </property>
   <property name="text">
    <string>困难</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	font: 75 20pt &quot;黑体&quot;;
background-color:rgb(170, 170, 255);
color:white;
border-radius:30px;
border:4px grey;
border-style:inset;
}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 26pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>请选择游戏难度：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
